auth_user_case_insensitive/__init__.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2015-2017 LasLabs Inc.
+# Copyright 2021 Open Source Integrators
```

Comment:
This is best removed: porting a module does is not considered original work, and does not grant copyrights.